src/cli/upgrade.rs

```diff
@@ -179,6 +179,7 @@ pub async fn execute(args: Args) -> miette::Result<()> {
             &args.specs.feature,
             &[],
             false,
+            &None,
```

Comment:
Upgrading a pypi package will change its location in the `pyproject.toml` manifest, if it is in a non-default location. I think this should be considered a bug. This is already the case in main, so I did not address the issue here. If that's OK, I propose to open an issue specifically for this?

Reply:
Yes this looks like a bug!

Reply:
See [here](https://github.com/prefix-dev/pixi/issues/2470)